describealign.py

```diff
@@ -1105,7 +1104,7 @@ def combine_gui(video_files, audio_files, config_path):
   print_queue = multiprocessing.Queue()
   
   settings = read_config_file(config_path)
-  settings.update({'display_func':print_queue.put, 'yes':True})
+  settings.update({'yes':True})
```

Comment:
Nah display_func was being passed to `combine` to allow logging to the console with `display`, but with the use of redirect_stdout/redirect_stderr, all calls to print() are being redirected to the GUI console. We can transition away from using `display`.

I switched to redirect_stdout because by default on Windows (with PyInstaller) sys.stdout is None, and static_ffmpeg calls sys.stdout.write directly in several places, which was initially crashing.